base/standard/src/main/java/org/openscience/cdk/stereo/StereoElementFactory.java

```diff
@@ -739,6 +824,63 @@ ExtendedTetrahedral createExtendedTetrahedral(int v, Stereocenters stereocenters
             return new ExtendedTetrahedral(focus, neighbors, winding);
         }
 
+        ExtendedCisTrans createExtendedCisTrans(List<IBond> dbs, Stereocenters stereocenters) {
+
+            // only applies to odd-counts
+            if ((dbs.size() & 0x1) == 0)
+                return null;
+
+            IBond   focus    = dbs.get(dbs.size()/2);
+            IBond[] carriers = new IBond[2];
+            int     config   = 0;
+
+            IAtom begAtom = dbs.get(0).getOther(getShared(dbs.get(0), dbs.get(1)));
+            IAtom endAtom = dbs.get(dbs.size()-1).getOther(getShared(dbs.get(dbs.size()-1), dbs.get(dbs.size()-2)));
```

Comment:
OK, so the passed List<IBond> when creating it must be sorted... I never remember which List implementations are guaranteed sorted and which not...

Reply:
I don't know what you mean, sorted? I think you mean *ordered* which all lists are.

Reply:
What I mean is when the createExtendedCisTrans() is called, the past List<> must be ordered cumulatively... sounds reasonable, but might be good to say that in that method explicitly...